DSCPullServerAdmin/src/CmdLets/SetDSCPullServerAdminRegistration.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Isam.Esent.Interop;
+using DSCPullServerAdmin.src.Models;
+using DSCPullServerAdmin.src.Helpers;
+using System.Linq;
+using System;
+using System.Text;
+
+namespace DSCPullServerAdmin.src.CmdLets
+{
+    [Cmdlet(VerbsCommon.Set,
+        "DSCPullServerAdminRegistration")]
+    [OutputType(typeof(void))]
+    public class SetDSCPullServerAdminRegistration : BaseCmdlet
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true)]
+        [Alias("Id")]
+        [ValidateNotNullOrEmpty()]
+        public string[] AgentId;
+
+        [Parameter(Mandatory = true)]
+        [ValidateNotNullOrEmpty()]
+        public string[] Configurations;
```

Comment:
you can most certainly include multiple configurationnames. ie partial configurations. The LCM must have know about them otherwise it won't set them. If the partial configs are defined in advanced, setting them in the db like this works just fine.